rust/avdschema/src/schema/int.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2025 Arista Networks, Inc.
+// Use of this source code is governed by the Apache License 2.0
+// that can be found in the LICENSE file.
+
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+use crate::schema::base::Base;
+
+use super::{
+    any::AnySchema,
+    base::{
+        convert_types::ConvertTypes, documentation_options::DocumentationOptions,
+        valid_values::ValidValues,
+    },
+};
+
+#[skip_serializing_none]
+#[derive(Debug, Clone, Default, PartialEq, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct Int {
+    pub min: Option<i64>,
+    pub max: Option<i64>,
+    #[serde(flatten)]
+    pub base: Base<i64>,
+    #[serde(flatten)]
+    pub convert_types: ConvertTypes,
+    #[serde(flatten)]
```

Comment:
<img width="842" alt="Screenshot 2025-03-14 at 18 47 17" src="https://github.com/user-attachments/assets/17a2fe21-1752-467c-ab49-1070bb117a5e" />

https://serde.rs/attr-flatten.html

seems like we are gettign it to work but just wanted to point out this warning